subprojects/gradle/test-project/src/main/kotlin/com/avito/test/gradle/InMemoryProjectGenerator.kt

```diff
@@ -1,15 +1,15 @@
-package com.avito.module.dependencies
+package com.avito.test.gradle
 
 import org.gradle.api.Project
 import org.gradle.api.plugins.JavaLibraryPlugin
 import org.gradle.testfixtures.ProjectBuilder
 
```

Comment:
Moved test fixtures for `ProjectBuilder` to `test-project` module.
There are lots of needed improvements in API to make them more reusable, but it's out of scope.